dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/strategy/HtmlPageTransformStrategy.java

```diff
@@ -0,0 +1,46 @@
+package com.dotmarketing.portlets.contentlet.transform.strategy;
+
+import static com.dotmarketing.portlets.fileassets.business.FileAssetAPI.MIMETYPE_FIELD;
+import static com.dotmarketing.portlets.fileassets.business.FileAssetAPI.TITLE_FIELD;
+import static com.dotmarketing.util.UtilMethods.isNotSet;
+
+import com.dotmarketing.beans.IconType;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.htmlpageasset.model.HTMLPageAsset;
+import com.dotmarketing.util.UtilHTML;
+import java.util.Map;
+import java.util.Set;
+
+public class HtmlPageTransformStrategy extends WebAssetStrategy<HTMLPageAsset> {
+
+    HtmlPageTransformStrategy(final TransformToolBox toolBox) {
+        super(toolBox);
+    }
+
+    @Override
+    public HTMLPageAsset fromContentlet(final Contentlet contentlet) {
+        return toolBox.htmlPageAssetAPI.fromContentlet(contentlet);
+    }
+
+    @Override
+    public Map<String, Object> transform(final HTMLPageAsset page, final Map<String, Object> map,
+            Set<TransformOptions> options)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'options' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)